pdk.gemspec

```diff
@@ -3,25 +3,26 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'pdk/version'
 
 Gem::Specification.new do |spec|
-  spec.name    = 'pdk'
+  spec.name = 'pdk'
   spec.version = PDK::VERSION
   spec.authors = ['Puppet, Inc.']
-  spec.email   = ['pdk-maintainers@puppet.com']
+  spec.email = ['pdk-maintainers@puppet.com']
 
-  spec.summary     = 'A key part of the Puppet Development Kit, the shortest path to better modules'
+  spec.summary = 'A key part of the Puppet Development Kit, the shortest path to better modules'
   spec.description = 'A CLI to facilitate easy, unified development workflows for Puppet modules.'
-  spec.homepage    = 'https://github.com/puppetlabs/pdk'
+  spec.homepage = 'https://github.com/puppetlabs/pdk'
 
-  spec.files         = Dir['CHANGELOG.md', 'README.md', 'LICENSE', 'lib/**/*', 'exe/**/*', 'locales/**/*']
-  spec.bindir        = 'exe'
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.files = Dir['CHANGELOG.md', 'README.md', 'LICENSE', 'lib/**/*', 'exe/**/*', 'locales/**/*']
+  spec.bindir = 'exe'
+  spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
   spec.required_ruby_version = '>= 2.1.9'
 
+  spec.add_runtime_dependency 'addressable', '2.6.0'
```

Comment:
wondering if this should be <= 2.6.0 ?

Reply:
The changelog and gemspec for addressable say it _should_ work for Ruby 2.1 anyway, so perhaps we don't need it at all? `addressable` depends on `public_suffix` which dropped ruby 2.1 support in 4.x (but ok in 3.x)

Reply:
I only added addressable here to force the version as the changelog generator was too permissive for addressable gem and caused this problem.

Reply:
Also now rake updated on the 27th.   So will need to force this to less than 13.0.0.

Reply:
The never-ending story dependency story :-(

Reply:
Is "addressable" really needed at runtime by the PDK gem? Is the `Gemfile` override sufficient instead?